inst/BS5/templates/content-citation-authors.html

```diff
@@ -1,18 +1,10 @@
 <div class="row">
   <div class="contents col-md-9">
+    <section class="section level2 authors-section">
```

Comment:
Yeah, I'd stick with divs for now, and we can consider if it's worth switching to `<sections>` later (and I doubt it's worth the effort)